src/utils/XRPL/account.ts

```diff
@@ -0,0 +1,63 @@
+import * as xrpl from "xrpl";
+import { Core } from "./core";
+import config from "../../../config";
+import { Wallet } from "xrpl";
+
+export interface IBalance {
+    value: string;
+    currency: string;
+}
+
+export class Account {
```

Comment:
Adding a comment before a class might be a good idea to explain why it is necessary

Reply:
I added comments to the classes.